contracts/lib/Organization.sol

```diff
@@ -0,0 +1,246 @@
+pragma solidity ^0.5.0;
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "./SafeMath.sol";
+import "./OrganizationInterface.sol";
+import "./IsWorkerInterface.sol";
+
+
+/**
+ * @title Organization contract.
+ *
+ * @notice The organization represents an entity that manages the
+ *         economy and therefore the Organization.sol contract holds all
+ *         the keys required to administer the economy.
```

Comment:
This organization contract will be used to upgrade core address, bounty and activation/deactivation of gateways. The comment looks odd.

Reply:
Updated